apps/site/src/app/about/TeamSection.tsx

```diff
@@ -21,11 +23,11 @@ const TeamSection = ({ team, members }: TeamSection) => {
 			<div className={styles.teamGrid}>
 				{members.map((member) => (
 					<TeamCard
-						key={member.name}
-						name={member.name}
+						key={member.person.name}
+						name={member.person.name}
 						position={member.position}
-						image={member.image}
-						linkedInUrl={member.linkedInUrl}
+						image={member.person.profilePic?.asset.url}
```

Comment:
Instead of retrieving the image url via the groq query, you can pass the image object into [@sanity/image-url](https://www.npmjs.com/package/@sanity/image-url).

This has several advantages:
- Respects the cropping the user has done in Sanity Studio
- Allows you to set the image format type and size for optimization
- Cleans up your groq query